.github/ISSUE_TEMPLATE/GITHUB_REQUEST_TEMPLATE.yaml

```
@@ -0,0 +1,52 @@
+---
+name: '⚙️ GitHub Request'
+description: 'Request an update to permissions/management/settings on the opensearch-project GitHub organization or its repositories.'
+title: '[GitHub Request]'
+labels: ['github-request']
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to submit GitHub Requset! Please note that the estimate time for the request to be resolved is 7-10 days from the time of the request.
+  - type: dropdown
+    attributes:
+      description: 'Provide request type'
+      label: 'What is the type of request?'
+      multiple: false
+      options:
+        - # Empty first option to force selection
+        - User Permission
+        - Organization Membership
+        - Repository Management
+        - Settings Update
+        - Secret Update
+        - GitHub Action
+        - Other
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      description: 'Describe the actions you are requesting (e.g., GitHub Actions in the <repository> take too long to allocate runners). If this is about a repository, please contact the corresponding maintainers before raising the issue.'
+      label: 'Details of the request'
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      description: 'Provide reasons for your request. For example, if you request to add a new maintainer to a repository, confirm that you have followed the steps in the [Becoming a Maintainer](https://github.com/opensearch-project/.github/blob/main/RESPONSIBILITIES.md#becoming-a-maintainer) guide, and provide the required materials and votes.'
+      label: 'Additional information to support your request'
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      description: 'Provide the timeline of your request. Requests typically require 7-10 business days to process. Specify your request timeline and whether or not it is an urgent request.'
+      label: 'When does this request need to be completed?'
+    validations:
+      required: true
+  - type: textarea
```

Seems like this is informative, what do you think about changing it to markdown?
```suggestion
  - type: markdown
```

---

Making it markdown will disappear after an issue is sent, I want to keep it as part of the issue description.

---

Can we at least drop the 'DO NOT DELETE' labeling?

Note; I think this would be better suite as a small GHA bot that adds a comment with the following:

> Thanks for your request, track the progress of your request on the project board: https://github.com/orgs/opensearch-project/projects/208/views/33